src/service/world-api-adapter.js

```diff
@@ -130,7 +192,21 @@ module.exports = function (config) {
         },
 
         /**
-        * Delete an existing world
+        * Delete an existing world.
+        *
+        *  **Example**
+        *
+        *      var gm = new F.service.World({ 
+        *           account: 'acme-simulations', 
+        *           project: 'supply-chain-game', 
+        *           group: 'team1' });
+        *      gm.create({ model: 'model.py' });
+        *           .then(function(world) {
+        *               gm.delete();
+        *           });
+        *
+        *  **Parameters**
+        * @param {object} `options` (Optional) Options object to override global options.
         *
         */
         delete: function (options) {
```

Comment:
This should take in an optional worldid as the first parameter @jaimedp - maybe for example if the admin is loading a bunch of worlds and wants to remove one of them